### PR TITLE
restructure workspace for publish

### PR DESCRIPTION
**Pull request by andylokandy**
- Update minstant
- Setup workspace
- add benchmark to readme

### CODE REVIEW

File: minitrace-jaeger/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+../LICENSE
```
Comment (zhongzc) [inline minitrace-jaeger/LICENSE:1]: ?

File: minitrace-jaeger/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+../README.md
```
Comment (zhongzc) [inline minitrace-jaeger/README.md:1]: ?

File: minitrace-jaeger/LICENSE (hunk 1 above)
Comment (andylokandy) [reply minitrace-jaeger/LICENSE:1]: These are symbolic links

File: minitrace/tests/lib.rs [hunk 3]
```diff
@@ -390,8 +390,6 @@ fn assert_graph(spans: Vec<SpanRecord>, expected_graph: &str) {
             result, expected_graph
         );
     }
-
-    assert_eq!(spans.iter().filter(|span| span.duration_ns == 0).count(), 0);
```
Comment (zhongzc) [inline minitrace/tests/lib.rs]: ？

Comment (andylokandy) [reply minitrace/tests/lib.rs]: duration by coarse time on mac is 0

Comment (andylokandy) [reply minitrace/tests/lib.rs]: oh, i've an idea

**Review by zhongzc — APPROVED** [review]

**Pull request merged**